### ISSUE TITLE
How to Extract Label Track

### ISSUE THREAD

**Issue by djsg** [davidavdav/audacity.py#6]
Hi,

Is this library able to extract Label Track?

**Comment by adabru** [comment]
You can use this tool: https://github.com/audacity/audacity-project-tools. Download and extract the release for your platform. Then run:

```
.\audacity-project-tools -print -analyze_project my_project.aup3
.\audacity-project-tools -print -extract-clips my_project.aup3
```

**Comment by davidavdav** [comment]
No, probably not.  But I suppose you can study the .xml and see if you can adapt the code.  Alternatively, use the tool `audacity-project-tools` quoted above, which probably makes this package obsolete.
